source/class/cv/ui/structure/tile/components/Chart.js

```diff
@@ -681,29 +736,60 @@ qx.Class.define('cv.ui.structure.tile.components.Chart', {
           .join('path')
           .style('mix-blend-mode', config.mixBlendMode)
           .attr('stroke', typeof config.color === 'function' ? p => config.color(p[0]) : null)
-          .attr('d', d => line(d[1]));
+          .attr('d', d => {
+            const curveName = this._dataSetConfigs[d[0]].curve || 'linear';
+            const func = lineFunctions[curveName] || lineFunctions.linear;
+            return func(d[1]);
+          });
       }
 
       // Add the area
       if (areaGroups.size > 0) {
-        const area = d3
-          .area()
-          //.defined(i => D[i])
-          .curve(config.curve)
-          .x(i => xScale(X[i]))
-          .y0(() => config.yRange[0])
-          .y1(i => yScale(Y[i]));
+        const areaFunctions = {};
+        for (let key of areaGroups.keys()) {
+          const curveName = this._dataSetConfigs[key].curve || 'linear';
+          if (!Object.prototype.hasOwnProperty.call(areaFunctions, curveName)) {
+            let curveFunction;
+            switch (curveName) {
+              case 'linear':
+                curveFunction = d3.curveLinear;
+                break;
+
+              case 'step':
+                curveFunction = d3.curveStep;
+                break;
+
+              case 'natural':
+                curveFunction = d3.curveNatural;
+                break;
+            }
+            if (curveFunction) {
+              // Construct a line generator.
+              areaFunctions[curveName] = d3
+                .area()
+                //.defined(i => D[i])
```

Comment:
Is the comment here on purpose or just a left over missed to delete?

Reply:
Those are all leftovers.